packages/jellyfish-transaction/src/script/index.ts

```diff
@@ -1,11 +1,12 @@
-export * from './bitwise'
```

Comment:
Reorganize this file like order in https://github.com/DeFiCh/jellyfish/issues/107

Reply:
It was good. Now it is not in correct order, probably is the result of GIT merge